config/locales/en/placements/providers/users.yml

```
@@ -8,14 +8,16 @@ en:
           change_organisation: Change organisation
         show:
           change_organisation: Change organisation
-          remove_user: Remove user
+          delete_user: Delete user
         destroy:
-          user_removed: User removed
+          user_removed: User deleted
```

```suggestion
          user_deleted: User deleted
```
Might also need changing in the controller.